core/src/test/java/org/elasticsearch/action/IndicesRequestIT.java

```diff
@@ -234,7 +235,7 @@ public void testUpdate() {
         client().prepareIndex(indexOrAlias, "type", "id").setSource("field", "value").get();
         UpdateRequest updateRequest = new UpdateRequest(indexOrAlias, "type", "id").doc("field1", "value1");
         UpdateResponse updateResponse = internalCluster().coordOnlyNodeClient().update(updateRequest).actionGet();
-        assertThat(updateResponse.isCreated(), equalTo(false));
+        assertThat(updateResponse.getOperation(), not(DocWriteResponse.Operation.CREATE));
```

Comment:
I'd prefer `assertEquals(DocWriteResponse.Operation.INDEX, updateResponse.getOperations());`.

I prefer equality here because it is a stronger test. If we have the information we may as well use it. I don't like using Hamcrest for strait equality because it can get overly complicated and I don't mind switching the argument order to match normal junit.